src/routes/user.route.js

```diff
@@ -9,15 +15,74 @@ router.post("/", async (request, response) => {
   try {
     const createdUser = await createUser(body);
     response.status(201);
+    response.json({
+      success: true,
+      data: { createdUser },
+    });
+  } catch (error) {
+    response.status(error.status || 400);
+
+    response.json({
+      success: false,
+      message: error.message,
+    });
+  }
+});
+
+router.get("/:id", verifyUser, async (request, response) => {
+  try {
+    const { params } = request;
+    const user = await getUser(params.id);
+    response.status(201);
     response.json({
       success: true,
       data: {

```

Comment:
Un poco redundate lo de userETC. Si no quieres un nesting mas puedes hacer:  
```suggestion
      data: user

```

Reply:
esta esta buena pero quería antes revisar si a futuro decidimos mandar solo alguna información del  user, porque si lo mandamos completo viene hasta la contraseña hasheada.

Reply:
Ah si, la contraseña no se manda. Wachate estos 2 approach entons, obviamente con lo que quieras regresar, yo nomas puse nomnbre y email.
1 approach :
```suggestion
      const { email, name } = user 
      data: {
           email,
           name
      }
```
En Javascript si la propiedad de un objeto y la variable que contiene su valor se llaman igual, solo se pone una vez.

2 approach:
Excluir pass
https://mongoosejs.com/docs/api.html#query_Query-select
![Screen Shot 2022-10-14 at 11 50 22](https://user-images.githubusercontent.com/74441510/195900154-95ad7267-8f9d-4266-aa97-e54cfec7aec9.png)

El chiste era omitir el userAlgo :)

Reply:
yo se pero lo queria dejar como placeholder en lo que vamos resolviendo la logica y ya despues mandar solo lo que necesitamos, o asi lo pensaba